addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CsrfTokenScanRule.java

```diff
@@ -54,6 +55,10 @@
 
     private static final String MESSAGE_PREFIX = "ascanbeta.csrftoken.";
     private static final int PLUGIN_ID = 20012;
+    private static final Map<String, String> ALERT_TAGS =
+            CommonAlertTag.toMap(
+                    CommonAlertTag.OWASP_2021_A05_SEC_MISCONFIG,
+                    CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG);
```

Comment:
This could arguably be a number of things: AuthZ? Access Control? Design?

Reply:
Theres no reason not to mave multiple mappings to a given Top Ten...
I'd say not design as it could just be a coding mistake.